chsdi/models/vector/stopo.py

```diff
@@ -197,11 +163,30 @@ class ShopProductGroupClass:
     scale = Column('scale', Integer)
     price = Column('price', Integer)
     release = Column('release', Integer)
+    data = Column('data', Integer)
+    isbn = Column('s_isbn', Text)
+    author = Column('s_author', Text)
     available = Column('available', Boolean)
     the_geom = Column(Geometry(geometry_type='GEOMETRY',
                                dimension=2, srid=21781))
 
 
+class SkitourenkarteMetadata (Base, ShopProductGroupClass, Vector):
+    __table_args__ = ({'schema': 'datenstand', 'autoload': False})
+    __tablename__ = 'view_gridstand_lkski_shop'
+    __bodId__ = 'ch.swisstopo.skitourenkarte-50.metadata'
+
+register('ch.swisstopo.skitourenkarte-50.metadata', SkitourenkarteMetadata)
```

Comment:
I assume here that `ch.swisstopo.skitourenkarte-50.metadata` as new table name and new attributes names.